src/include/ceph_fs.h

```diff
@@ -422,6 +423,7 @@ union ceph_mds_request_args_legacy {
 		__le32 max_entries;          /* how many dentries to grab */
 		__le32 max_bytes;
 		__le16 flags;
+               __le32 offset_hash;
```

Comment:
Is the idea that this is safe to add because other items in the union already had a greater size, and request structures are written with zeros in old clients, so we will always get a zero here from older clients?